activestorage/lib/active_storage/service.rb

```diff
@@ -121,6 +132,19 @@ def headers_for_direct_upload(key, filename:, content_type:, content_length:, ch
       {}
     end
 
+    def public_service?
+      !!@public_service
+    end
```

Comment:
`service.public_service?` is redundant. Let’s name this method `public?` and the corresponding option and instance variable `public:` and `@public`.